core/templates/components/summary-tile/exploration-summary-tile.directive.ts

```diff
@@ -227,3 +227,14 @@ angular.module('oppia').directive('explorationSummaryTile', [
       ]
     };
   }]);
+
+import { Directive, ElementRef, Injector } from '@angular/core';
```

Comment:
What is going on here?  I didn't get. We are changing the directives in the collection-player-page right?

Reply:
So, collection-footer.component.html needed exploration-summary-tile. So, @srijanreddy98 suggested I needed to wrap it as a directive. For reference: https://github.com/oppia/oppia/pull/9237/files